graphql/rcos/projects/get_by_id.graphql

```diff
@@ -2,8 +2,8 @@ query Project($id: Int!) {
   project: projects_by_pk(project_id: $id) {
     project_id
     description
-    stack
-    repository_urls
+    # stack
```

Comment:
Any reason to comment out rather then delete

Reply:
I still need to implement them, so they're just a reminder for me